### DISCUSSION TITLE
Unable to get external IP error

### DISCUSSION THREAD

**Discussion by emartinez11** [GameServerManagers/LinuxGSM#4013] [Q&A]
Im sure im just overlooking something, but i get this error on each server I install and start. It doesnt stop the server in any way and they work just fine.. just curious how to fix this error

**Comment by utahbmxer** [comment]
I was having the same issue and found that my pi-hole was blocking the domain while the script was trying to call https://api.ipify.org.  Added to the whitelist and problem solved.  Not sure if this is your case, but something to check.

**Reply by DaVinniCode** [reply]
Dang nice work seeing it was your pi-hole.  I rarely need to check the blocked queries, but there it was...   Good find in my case!
